ai-safety-debate/precompute_debate_results.py

```diff
@@ -62,9 +81,9 @@ def get_debate_results(start_point, batch_size, N_train, N_to_mask, judge_path):
 
     args = parser.parse_args()
 
-    results = np.zeros((args.N_train, 10))
-    batch_size = math.ceil(args.N_train / args.N_threads)
-    start_points = [i * batch_size for i in range(args.N_threads)]
+    results = np.zeros((args.N_train - args.start_sample, 10))
```

Comment:
If I interpret this right, N_train isn't the Number of train samples you run anymore, but the index of the train sample you run up to. Maybe rename it to end_sample or train_until or something like that, instead?

Reply:
Yeah I wanted it to still be the number of samples, but it is just wrong :laughing: 
Will fix this and also add the option to precompute debates on a test set, which is what we probably want to evaluate.